### PR TITLE
Update Qt tutorials with VTK9

### LINKED ISSUE #5004: [visualization] Update QT tutorials to support vtk9
**Is your feature request related to a problem? Please describe.**
With PCL 1.12 you added support for vtk9. Now with the new version the QT tutorials don't compile anymore.
`#include >QVTKWidget.h` throws an error.

**Context**

Trying to update my code to VTK9 and PCL 1.12.

**Expected behavior**

QT tutorials compile.

**Current Behavior**

QT tutorials don't compile due to deprecated QVTKWidget.

**Describe the solution you'd like**

Update the tutorials.

**Describe alternatives you've considered**

The tutorials helped me a lot getting started. It would be nice to have a boilerplate for getting started with QT, VTK9 and PCL. I tried to follow the changes in the [apps] from https://github.com/PointCloudLibrary/pcl/pull/4262 but I am a bit lost. The tutorials were small and easy to read for me.

**Additional context**

Of course thanks for all the time you invested in this awesome project.

### PR DESCRIPTION
**Pull request by larshg**
Closes #5004.

### CODE REVIEW

**Comment by larshg** [comment]
Closes also this one #3851

File: doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt [hunk 1]
```diff
@@ -27,3 +27,9 @@ set(project_SOURCES main.cpp pclviewer.cpp)
 add_executable(${PROJECT_NAME} ${project_SOURCES})
 
 target_link_libraries(${PROJECT_NAME} ${PCL_LIBRARIES} Qt5::Widgets)
+
+#TODO: Update when CMAKE 3.10 is available
```
Comment (SunBlack) [inline doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt:31]: Isn't it the minimum required version already?

Comment (larshg) [reply doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt:31]: Yes, but I wrote this as part of the vtk 9 branch. Just havent got around to update it after the bump to 3.10 :-)

Comment (larshg) [reply doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt:31]: Yes. And yes 😄

**Comment by mvieth** [comment]
Since you changed pclviewer.h and pclviewer.cpp, you should adapt the line references in qt_visualizer.rst and qt_colorize_cloud.rst. Also maybe undo the changes in CMakeLists.txt?

**Comment by larshg** [comment]
@mvieth I have made the requested changes, thanks for reminding.

**Review by mvieth — APPROVED** [review]
Looks good

**Comment by SunBlack** [comment]
Tried it on our CI: Some combinations which worked before, are now throwing compile errors

```
-- The imported target "vtkRenderingPythonTkWidgets" references the file
   "/usr/lib/x86_64-linux-gnu/libvtkRenderingPythonTkWidgets.so"
but this file does not exist.  Possible reasons include:
* The file was deleted, renamed, or moved to another location.
* An install or uninstall procedure did not complete successfully.
* The installation package was faulty and contained
   "/usr/lib/cmake/vtk-6.3/VTKTargets.cmake"
but not all the files it references.

-- The imported target "vtk" references the file
   "/usr/bin/vtk"
but this file does not exist.  Possible reasons include:
* The file was deleted, renamed, or moved to another location.
* An install or uninstall procedure did not complete successfully.
* The installation package was faulty and contained
   "/usr/lib/cmake/vtk-6.3/VTKTargets.cmake"
but not all the files it references.
...
nvcc fatal   : The version ('60000') of the host compiler ('clang') is not supported
CMake Error at pcl_cuda_segmentation_generated_connected_components.cu.o.Release.cmake:219 (message):
  Error generating
  /../pcl/build/cuda/segmentation/CMakeFiles/pcl_cuda_segmentation.dir/src/./pcl_cuda_segmentation_generated_connected_components.cu.o


cuda/segmentation/CMakeFiles/pcl_cuda_segmentation.dir/build.make:63: recipe for target 'cuda/segmentation/CMakeFiles/pcl_cuda_segmentation.dir/src/pcl_cuda_segmentation_generated_connected_components.cu.o' failed
make[2]: *** [cuda/segmentation/CMakeFiles/pcl_cuda_segmentation.dir/src/pcl_cuda_segmentation_generated_connected_components.cu.o] Error 1
CMakeFiles/Makefile2:5174: recipe for target 'cuda/segmentation/CMakeFiles/pcl_cuda_segmentation.dir/all' failed
make[1]: *** [cuda/segmentation/CMakeFiles/pcl_cuda_segmentation.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Comment by larshg** [comment]
I guess it was to the other PR that post was intended, @SunBlack ?

**Comment by SunBlack** [comment]
Well not, it is the correct MR (checked the CI log again, to be sure I didn't used the wrong branch):
```
Checking out Revision 0ee3a7d29504ecfae7fbdf0bb5143267e1a45964 (refs/remotes/origin/VTK9SupportTutorials)
Commit message: "Update qt_visualizer and line numbers"
```

**Comment by larshg** [comment]
> Well not, it is the correct MR (checked the CI log again, to be sure I didn't used the wrong branch):
> 
> ```
> Checking out Revision 0ee3a7d29504ecfae7fbdf0bb5143267e1a45964 (refs/remotes/origin/VTK9SupportTutorials)
> Commit message: "Update qt_visualizer and line numbers"
> ```

Okay :smile: 
This PR doesn't change anything about cuda compiling, so I find it odd. Could you have updated the Clang compiler, since thats what its complaining about?

**Comment by SunBlack** [comment]
Started direct after this failing build a build of the pcl master with same settings and there I had no compile issue. So indeed, this is a strange error.

**Pull request merged**